l10n_de_location_nuts/readme/INSTALL.rst

```diff
@@ -0,0 +1,3 @@
+To install this addon, you need 'requests' python module:
```

Comment:
These instructions can be removed, as `requests` is directly an Odoo requirement.